contracts/l1-deployer-cli.sh

```diff
@@ -248,6 +248,9 @@ get_chain_params() {
     elif [[ "$chain" == "holesky" ]]; then
         chain_id=17000
         deploy_contract="DeployHolesky"
+    elif [[ "$chain" == "hoodi" ]]; then
+        chain_id=23118
```

Comment:
I think it's this chain id, correct me if I'm wrong
```suggestion
        chain_id=560048
```

Reply:
yeah thats correct I pulled the wrong one and stuck with it